.cbmc-batch/jobs/aws_cryptosdk_keyring_trace_record_init_clone/aws_cryptosdk_keyring_trace_record_init_clone_harness.c

```diff
@@ -24,7 +24,10 @@ void aws_cryptosdk_keyring_trace_record_init_clone_harness() {
     struct aws_allocator *alloc = can_fail_allocator();      /* Precondition: alloc must be non-null */
 
     source_record.wrapping_key_namespace = ensure_string_is_allocated_bounded_length(MAX_STRING_LEN);
-    source_record.wrapping_key_name      = ensure_string_is_allocated_bounded_length(MAX_STRING_LEN);
+    __CPROVER_assume(source_record.wrapping_key_namespace != NULL);
+
+    source_record.wrapping_key_name = ensure_string_is_allocated_bounded_length(MAX_STRING_LEN);
+    __CPROVER_assume(source_record.wrapping_key_name != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Same as above.